Setting up a sentry node (#171)
w3f/polkadot-wiki: docs/polkadot/build/hackathon.md

# Hackathon

> As one of our objectives in the Web3 Foundation is pushing Web3 technology stack, we would like to support anyone/team who wants to organize a hackathon for getting more developers to understand about Polkadot/Substrate in your city.

Reviewer: ```suggestion
As one of our objectives in the Web3 Foundation is pushing the [Web3 technology stack](http://wiki.web3.foundation/en/latest/tech_stack/tech_stack_overview/), we would like to support any individual or team who wants to organize a hackathon that gets more developers to understand Polkadot/Substrate in your city.
```

> We are not only providing technical resources for you, but we also can have someone from our team to give a hands-on workshop/presentation about the technology to those participants get started hacking on it in a much quicker way.

Reviewer: ```suggestion
We are not only providing technical resources for you, but we can have someone from our team give a hands-on workshop/presentation about the technology to help participants get started hacking on it more quickly.
```

## Ideas 

- NFT
- Bonded Tokens
- New governance modules with different styles of voting & democracy
- Upgrade the TCR module
- IPFS integration
- Decentralized identity solutions
- Oracle module
- Implement Generalized State Channel/Plasma Chain
- Decentralized Exchange
- Tools for monitoring validator performance
- dApps across different parachains

## Why you should host a hackathon?

- Love to deliver the vision of Web3 

> - Learn the leading-edge technology

Reviewer: ```suggestion
- Learn the bleeding-edge technology
```

> - Believe our technology stack is suitable for developers to build their applications

Reviewer: ```suggestion
- Believe the Web3 technology stack is suitable for developers to build their applications
```

- Like to make connections

## How do hackathons differ from workshops?

> Hackathon is aiming for those participants to deliver their prototype to demonstrate their ideas to reality in a short period (few hours - one day), while the workshop is letting them get familiar with the tooling and knowing more about what are the resources currently available and how they can make use of it.

Reviewer: ```suggestion
Hackathon is aimed at participants who want to deliver their prototype and demonstrate their ideas to reality in a short period (few hours - one day). Workshops are more for getting developers familiar with the tooling and understanding more about what are the resources currently available and how they can make use of it.
```

## How can you run a hackathon in your city?
1. Submit your application (Coming soon.)
2. Review
3. Decision
4. Follow up

> ## What we can help you?

Reviewer: ```suggestion
## How we can help you
```
